### PR TITLE
docs(examples): Add "How To" generate env vars and restructure .env.site comments.

### PR DESCRIPTION
**Pull request by VancheeZze**
## Changes
- Add documentation on how to generate env variables to the example sites.
- Restructure `.env.site` comments style to follow the same pattern.
- Added `build:env-vars` script to the root package.json which leverages lerna to run `build:env-vars` script in all packages.

## Related Issues
Relates to: https://github.com/johnsonandjohnson/Bodiless-JS/issues/50

### CODE REVIEW

File: examples/test-site/.env.site [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
+# The Google Fonts plugin is enabled by default.
+# To disable it - uncomment the following line.
 # GOOGLE_FONTS_ENABLED='0'
 
-# GTM is enabled on Bodiless example site by default and is rendered in the static site.
-# For GTM activation/disablement, you should set the following variables in the .env.production file that controls static site.
-
-# GTM Disablement
-# GTM is enabled by default. To disable GTM, uncomment the following line.
+# GTM is enabled by default.
```
Comment (hvanyo) [inline examples/test-site/.env.site]: GTM is disabled by default

File: examples/test-site/.env.site [hunk 2]
```diff
@@ -1,30 +1,31 @@
...
+# To disable it - uncomment the following line
```
Comment (hvanyo) [inline examples/test-site/.env.site]: to enable it
next line set to = "1"

File: examples/test-site/.env.site (hunk 1 above)
Comment (VancheeZze) [reply examples/test-site/.env.site]: Updated.

File: examples/test-site/.env.site (hunk 2 above)
Comment (VancheeZze) [reply examples/test-site/.env.site]: Done

File: examples/test-site/.env.site (hunk 1 above)
Comment (VancheeZze) [reply examples/test-site/.env.site]: I think it just this section is never gets updated. If we look into "Files Changed" then you will see the changes: https://github.com/johnsonandjohnson/Bodiless-JS/pull/83/files#diff-9d823bf4f52f8e81b8d295475c23c541R19

**Pull request merged**